games-arcade/wormux/wormux-master.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Very nice game"
+DESCRIPTION="Arcade game based on Worms"
+HOMEPAGE="http://wormux.org"
+COPYRIGHT="<a-team>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/a-team/wormux/archive/master.zip"
+ARCHITECTURES="x86_64 x86_gcc2 x86"
```

Comment:
Add a newline before this line.

Reply:
It should be: ARCHITECTURES="x86_gcc2 x86 x86_64"
Does it builds on every arch? If untested, add a question mark before them, eg: ?x86_gcc2
If known broken, add an exclamation mark, like: !x86_gcc2